Sources/Compass.swift

```diff
@@ -18,21 +18,21 @@ public struct Compass {
 
   public static var routes = [String]()
 
-  public static func parse(url: NSURL, payload: Any? = nil) -> Location? {
-    let path = url.absoluteString.substringFromIndex(scheme.endIndex)
+  public static func parse(_ url: URL, payload: Any? = nil) -> Location? {
```

Comment:
I would leave it like `public static func parse(url: NSURL, payload: Any? = nil)`. Thought? @onmyway133 @zenangst @RamonGilabert ?